sound/soc/sof/sof-client.c

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
+//
+
+#include <linux/device.h>
+#include "sof-client.h"
+
+static LIST_HEAD(client_list);
+static DEFINE_MUTEX(sof_client_mutex);
+
+int sof_client_drv_register(const struct sof_client_drv *drv, struct device *dev)
```

Comment:
nitpicking: I would find the reverse argument order more natural, but maybe it's just me.

Reply:
yes, now that you mention it, it is a bit odd to have it the other way around. fixed now.